front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx

```diff
@@ -30,7 +30,7 @@ const SensorDeviceType = ({ children, ...props }) => (
     <td>{props.deviceFeature.name}</td>
     {SPECIAL_SENSORS.indexOf(props.deviceFeature.category) === -1 && (
       <td class={cx('text-right', { 'text-nowrap': props.deviceFeature.last_value !== null })}>
-        {props.deviceFeature.last_value !== null && props.deviceFeature.last_value}
+        {props.deviceFeature.last_value !== null && props.deviceFeature.last_value.toFixed(2)}
```

Comment:
Sets the number of digits to the stored or retrieved value rather than the displayed one.
Not all sensors require 2 digits.

Reply:
I fix number of digits in value store now